tide_site.module

```diff
@@ -662,3 +662,38 @@ function tide_site_search_api_index_items_alter(IndexInterface $index, array &$i
 function tide_site_tide_api_jsonapi_custom_query_parameters_alter(&$custom_params) {
   $custom_params[] = 'site';
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function tide_site_form_views_exposed_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if (Drupal::routeMatch()->getRouteName() == 'system.admin_content' && _has_target_site_fields()) {
+    $form['#attached']['library'][] = 'tide_site/tide_site_layout';
+    $primary_sites_term_options = [];
+    $storage = Drupal::entityTypeManager()->getStorage('taxonomy_term');
```

Comment:
@vincent-gao fully-qualified namespace `\ Drupal`.

Reply:
@vincent-gao and you can use `TideSiteHelper::getAllSites()` instead.

Reply:
ok. solved.